crates/propolis-api-types/src/instance_spec/components/backends.rs

```diff
@@ -47,6 +47,12 @@ pub struct FileStorageBackend {
 
     /// Indicates whether the storage is read-only.
     pub readonly: bool,
+
+    /// Block size of the backend
+    pub block_size: u32,
+
+    /// Optional worker threads for the file backend, exposed for testing only.
+    pub workers: Option<usize>,
```

Comment:
Does the spec get grumpy if you use a NonzeroUsize here?

Reply:
It seems to work, so I've updated it all to use `NonZeroUsize`